### PR TITLE
English: Add more stuff to optional post install stuff

### PR DESCRIPTION
**Pull request by halal-beef**

### CODE REVIEW

**Comment by halal-beef** [comment]
@halal-beef @bibarub approve changes wen

File: guide/English/optional-postinst-stuff-en.md [hunk 1]
```diff
@@ -37,3 +37,102 @@
...
+## Finished!
+
+
+
+
+### Disabling USB Host mode
+
+> **Warning** Any non powered hub will STOP working!
+
+> **Warning** Make sure these steps are done on the POCO X3 Pro running windows!
```
Comment (bibarub) [inline guide/English/optional-postinst-stuff-en.md:124]: Windows (capital W)

File: guide/English/optional-postinst-stuff-en.md [hunk 2]
```diff
@@ -37,3 +37,102 @@
...
+
+```adb shell```
+
+###### Finding the modem folder
+
+> Take note of the random data in that folder name (after qcremotefs8150)
+
+```ls /win/Windows/System32/DriverStore/FileRepository/ | grep qcremotefs8150```
+
+###### Delete any previous modem files if they exist
```
Comment (bibarub) [inline guide/English/optional-postinst-stuff-en.md:75]: no need, dd will overwrite it

Comment (halal-beef) [reply guide/English/optional-postinst-stuff-en.md:75]: Ok

**Pull request merged**